RestaurantReservation.Db/Repositories/CustomerRepository.cs

```diff
@@ -0,0 +1,82 @@
+﻿using Microsoft.EntityFrameworkCore;
+using RestaurantReservation.Db.Models;
+
+namespace RestaurantReservation.Db.Repositories
+{
+    public class CustomerRepository : ICustomerRepository
+    {
+        private RestaurantReservationDbContext _context;
+
+        public CustomerRepository(RestaurantReservationDbContext context)
+        {
+            _context = context;
+            _context.Database.EnsureCreatedAsync().Wait();
+        }
+
+        public async Task<int> CreateAsync(CustomerDTO newCustomer)
+        {
+            if (newCustomer.Id < 0)
+            {
+                throw new Exception("Id property can't be negative.");
+            }
+            else if (await CustomerExistsAsync(newCustomer.Id))
```

Comment:
else is not needed, since `throw` will do the job and kick you out from the method and not continue, this applies to the rest of the methods here please